genesis/genesis.go

```diff
@@ -21,7 +21,7 @@ type Genesis struct {
 const defaultGasLimit = 30_000_000
 
 // Commit assumes the application has not been initialized and that the block store is empty.
-func (g *Genesis) Commit(ctx context.Context, app monomer.Application, blockStore store.BlockStoreWriter) error {
+func (g *Genesis) Commit(ctx context.Context, app monomer.Application, blockStore store.BlockStoreWriter, ethStateRoot []byte) error {
```

Comment:
prefer descriptive types
```suggestion
func (g *Genesis) Commit(ctx context.Context, app monomer.Application, blockStore store.BlockStoreWriter, ethStateRoot common.Hash) error {
```